opl/cluster_read.py

```diff
@@ -398,6 +434,7 @@ def measure(self, ri, name, **kwargs):
     "env_variable": EnvironmentPlugin,
     "command": CommandPlugin,
     "copy_from": CopyFromPlugin,
+    "count_line": CountLinePlugin,
```

Comment:
To register plugin in a way that it will be used for correct config file entries (see https://github.com/redhat-performance/opl/blob/df1b3555a43c37e15c86441081fe07286df7a499/opl/cluster_read.py#L495) you need to use `"log_source_command": CountLinePlugin,` IIRC (not sure here, please try).